docs/advanced/routing/external-routing.md

```diff
@@ -0,0 +1,100 @@
+# External Routing
+
+It is possible to resolve routes dynamically based on platform data. Daffodil enables this behavior with [`@daffodil/external-router`](../../../libs/external-router/README.md).
+
+## Getting Started
+
+### Configuration
+
+`@daffodil/external-router` needs information about how to construct the route. This info can be provided as an argument to [`DaffExternalRouterModule#forRoot`](../../../libs/external-router/src/external-router.module.ts).
+
+#### Insertion Strategy
+
+If routes need to be added at specific points in the router config, the `@daffodil/external-router` configuration exposes [`DaffTypeRoutePair#insertionStrategy`](../../../libs/external-router/src/external-router.module.ts). The insertion strategy can be used to customize the behavior for adding routes.
+
+> See [`@daffodil/external-router` configuration](../../../libs/external-router/guides/configuration.md).
+
+### Route Capture
```

Comment:
Please add a stackblitz example.

Reply:
Do you mean instead of the example or in addition to?

Reply:
In addition to. I'd like to be able to see a working example of what you intend from reading the doc.

Reply:
I think we would need to merge some of these PRs and do a release before anything will actually run.

Reply:
Can you do a contrived example rather than examples with resolvers from other packages?

Reply:
You mean with a stubbed resolver or something like that?

Reply:
Yes